src/hooks/useGitOpsColumns.tsx

```diff
@@ -0,0 +1,57 @@
+import { TableColumn } from "@openshift-console/dynamic-plugin-sdk";
+import { sortable } from "@patternfly/react-table";
+import * as classNames from 'classnames';
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { GitOpsAppGroupData } from "src/components/utils/gitops-types";
```

Comment:
Use .. ?

Reply:
Will update it to: import { GitOpsAppGroupData } from "../components/utils/gitops-types";